### ISSUE TITLE
NG05100: Providers from the `BrowserModule` have already been loaded. If you need access to common directives such as NgIf and NgFor, import the `CommonModule` instead.

### ISSUE THREAD

**Issue by joazco** [digisolu/ngx-dynamic-json-form#2]
**Describe the bug**
I want to integrate ngx-dynamic-json-form into my Angular library. I've also incorporated a Storybook into my library, and when I try to create a component that uses ngx-dynamic-json-form, I encounter this error.

**To Reproduce**

`form-basic.component.ts`

```ts
import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
import { FormGroup } from '@angular/forms';
import {  FormField } from 'ngx-dynamic-json-form-core';


@Component({
  selector: 'dse-form-basic',
  templateUrl: './form-basic.component.html',
  styleUrls: ['./form-basic.component.scss'],
  changeDetection: ChangeDetectionStrategy.OnPush,
})
export class FormBasicComponent {
  public form: FormGroup = new FormGroup({});

  public fields: FormField[] = [
    {
      type: "select",
      key: "type",
      label: "Type",
      options: [
        { label: "Language", value: "1" },
        { label: "Food", value: "2" },
        // ...
      ],
      onChange: () => {
        if (this.form?.get("type")?.value === "1") {
          this.form?.get("language")?.enable();
        } else {
          this.form?.get("language")?.disable();
        }
      },
    },
    {
      type: "select",
      key: "language",
      label: "Languages",
      options: [
        { label: "Language 1", value: "1" },
        // ...
      ],
      disabled: true,
    },
  ];

  public setForm(form: FormGroup): void {
    this.form = form;
  }
}

```

`form-basic.component.html`

```html
<ngx-dynamic-json-form
  [fields]="fields"
  (getForm)="setForm($event)"
></ngx-dynamic-json-form>
```

`form-basic.module.ts`

```ts
import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { FormBasicComponent } from './form-basic.component';
import { FormModule } from '../form.module';
import { NgxDynamicJsonFormMaterialModule } from 'ngx-dynamic-json-form-material';



@NgModule({
  declarations: [FormBasicComponent],
  imports: [
    CommonModule,
    NgxDynamicJsonFormMaterialModule.forRoot()
  ],
  exports: [FormBasicComponent]
})
export class FormBasicModule { }
```

`form-basic.stories.ts`

```ts
import {
  moduleMetadata,
  type Meta,
  type StoryObj,
  applicationConfig,
} from '@storybook/angular';
import {
  BrowserAnimationsModule,
  provideAnimations,
} from '@angular/platform-browser/animations';

import { importProvidersFrom } from '@angular/core';
import { MatNativeDateModule } from '@angular/material/core';
import { NgxDynamicJsonFormModule } from 'ngx-dynamic-json-form-core';
import { FormBasicComponent } from '../../lib/form/form-basic/form-basic.component';
import { FormBasicModule } from '../../lib/form/form-basic/form-basic.module';

const meta: Meta<FormBasicComponent> = {
  title: 'Form/Basic',
  component: FormBasicComponent,
  decorators: [
    applicationConfig({
      providers: [
        importProvidersFrom(BrowserAnimationsModule),
        provideAnimations(),
        importProvidersFrom(MatNativeDateModule),
      ],
    }),
    moduleMetadata({
      imports: [NgxDynamicJsonFormModule, FormBasicModule]
    }),
  ],
};
export default meta;
type Story = StoryObj<FormBasicComponent>;

export const Example: Story = {};

```

**Comment by digisolu** [comment]
Sorry for the late reply, the problem should be fixed in `Version: 1.0.4`.

There was an old Dependency on the `BrowserAnimationModule`.

**Comment by digisolu** [comment]
closed, was fixed in 1.0.4
